tests/data/files.csv

```diff
@@ -1,3 +1,6 @@
 url,file_type,md5_checksum
 test.txt,TXT,14758f1afd44c09b7992073ccf00b43d
-test.txt,TXT,definitelynottherightmd5checksum
+test.txt,TIFF,14758f1afd44c09b7992073ccf00b43d
```

Comment:
Nit: I see that file_type is all capitalized, should this be case agnostic when we are actually using it to determine the tests to run per file?

Reply:
Comparisons are made after making everything lowercase ([source](https://github.com/Sage-Bionetworks-Workflows/py-dcqc/blob/b21007d1fc57515abb1e0ef7845e66c23a86c425/src/dcqc/file.py#L43)).